### ISSUE TITLE
Categorize with OS Support

### ISSUE THREAD

**Issue by Owlz** [onethawt/idaplugins-list#3]
It'd be helpful if these were categorized by what OS the plugin supports. For instance, a majority of these are compiled for Windows. Only a few are compiled for Linux (or support compilation for it). Even fewer support OS X.

**Comment by onethawt** [comment]
That would definitely be helpful. I suppose I could go through them one afternoon and see whether they are cross-platform, but I'm not testing each. I'm also assuming that for the most part IDA Python plugins are cross-platform unless they're using the win32api.

Any help with this would be appreciated.

**Comment by vanhauser-thc** [comment]
Also it would be super helpful to give them an "supports IDA 7.x", "require IDA 7.x" or "requires IDA < 7.0" tag while you are at it (which would save time :) )

thanks for the list by the way, I appreciate the effort.

**Comment by onethawt** [comment]
I completely agree with you. Unfortunately the majority of plugins aren't exactly well maintained and I assume that most of the older Python based plugins wont run on 7.

The whole reorganization has been sitting in the queue for a long time, I suppose I can start chipping away at it this weekend a few at a time.

**Comment by hairlessbear** [comment]
Instead of trying to keep track of compatibility (especially since compatible with 7.0 doesn't guarantee compatible with 7.1, and vice versa), what about adding in the date of the last update to the plugin? That should make it obvious when plugins have been abandoned as well as showing which ones are more likely to still be maintained.

Although that info could be embedded into the current format, I think it might be clearer as a table. Something like this:


Plugin | Description | Last Updated
------- | ------------ | ---------------
[3DSX Loader](https://github.com/0xEBFE/3DSX-IDA-PRO-Loader) | IDA PRO Loader for 3DSX files | 3/1/2015
[Adobe Flash disassembler](https://hex-rays.com/contests/2009/SWF/ReadMe.txt) | The 2 plugins present in this archive will enable IDA to parse SWF files, load all SWF tags as segments for fast search and retrieval, parse all tags that can potentially contain ActionScript2 code, discover all such code(a dedicated processor module has been written for it) and even name the event functions acording to event handled in it (eg. OnInitialize). [Download](https://hex-rays.com/contests/2009/SWF/swf.zip) | 2009
[alleycat](https://github.com/devttys0/ida/tree/master/plugins/alleycat) | Finds paths to a given code block inside a function; finds paths between two or more functions; generates interactive call graphs; fully scriptable | 8/17/2016
[Amnesia](https://github.com/duo-labs/idapython) | Amnesia is an IDAPython module designed to use byte level heuristics to find ARM thumb instructions in undefined bytes in an IDA Pro database. Currently, the heuristics in this module find code in a few different ways. Some instructions identify and define new code by looking for comon byte sequences that correspond to particular ARM opcodes. Other functions in this module define new functions based on sequences of defined instructions. | 11/19/2017

What do you think? If you like the idea, I'm willing to do the grunt work to add the info and reorganize the page in this way.

**Comment by onethawt** [comment]
That would be fantastic!

**Comment by hairlessbear** [comment]
Sweet! I've started working on it on my fork. I'll get it done some time in the next couple weeks and submit a PR. While I'm at it, I'll make corrections/updates to the entries as needed.
